docker/pyproject.toml

```diff
@@ -22,4 +23,4 @@ isort = "^5.9.1"
 pylint = "^2.8.3"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
\ No newline at end of file
+build-backend = "poetry.core.masonry.api"
```

Comment:
cc @bojiang - `./docker` is a separate python project, make sure to update its dependency when making changes